frontend/src/concepts/pipelines/content/tables/pipeline/PipelinesTableRow.tsx

```diff
@@ -62,6 +70,7 @@ const PipelinesTableRow: React.FC<PipelinesTableRowProps> = ({
             id={pipeline.pipeline_id}
             isChecked={isChecked ? true : selectedVersionLength !== 0 ? null : false}
             onToggle={onToggleCheck}
+            isDisabled={disableDelete}
```

Comment:
Need to add a tooltip here -- maybe make the tooltip string a constant and provide it as a wrapper? We need to show why it's disabled from a check.